servant-server/src/Servant/Server.hs

```diff
@@ -17,6 +17,8 @@ module Servant.Server
   , -- * Handlers for all standard combinators
     HasServer(..)
   , Server
+  , EmptyAPIServer
```

Comment:
Dropping API from the other names sounds good to me; but I think I'd keep `EmptyAPI` over `Empty`, as it is more explicit.